addons/mail/models/mail_link_preview_message.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.mail.tools.discuss import Store
+
+
+class LinkPreviewMessage(models.Model):
+    _name = "mail.link.preview.message"
+    _inherit = "bus.listener.mixin"
+    _description = "Link between link previews and messages"
+
+    link_preview_id = fields.Many2one("mail.link.preview", index=True)
+    message_id = fields.Many2one("mail.message")
+    is_hidden = fields.Boolean(default=False)
```

Comment:
False is the default
```suggestion
    is_hidden = fields.Boolean()
```

Reply:
It is SO SAD that proper typing doesn't give me a hint that the default value is `False`…
![image](https://github.com/user-attachments/assets/f5ff7ed5-385e-4821-a957-7238a872979e)

Reply:
So typing is necessary because in almost 5 years at Odoo you don't know that a boolean field is Falsy by default ? This really makes me think typing is not the issue here.

Reply:
I don't fill my brain with useless information that my tools should provide. It's not just help for me, it's help for everyone, newbies and dinosaurs alike.

Reply:
I could, but not to software engineers. I like to talk with reasonable people…

Reply:
You are going to feel alone indeed.